tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -284,7 +285,7 @@ object TastyFormat {
    * compatibility, but remains backwards compatible, with all
    * preceeding `MinorVersion`.
    */
-  final val MinorVersion: Int = 0
+  final val MinorVersion: Int = 1
```

Comment:
The new policy for updating Tasty while we are still in RC should be to instead bump the `ExperimentalVersion`, then for the first final release we reset `ExperimentalVersion` to 0. (then the next time we change Tasty after 3.0.0 increase both `ExperimentalVersion` and `MinorVersion`)

Reply:
I don't think that can work. Experimental versions are incompatible with each other. So if I bump the experimental version to 1, it means that nobody else can bump it further, without being incompatible. Or not?

Reply:
The ExperimentalVersion is already 1 currently, it is only set to 0 for final releases (that is non-nightly, non-milestones, non-rc releases), for everything else we bump experimental every time we make a change, which makes us incompatible with previous experimental versions (which is fine since they were never part of a final release) but not previous non-experimental versions (which is what we need to keep supporting until the day we break tasty).

Reply:
OK, but that means we need to recompile everything. It's like a major version bump for us, right?

Reply:
Until 3.0.0 final, yes, it's a like a major version bump.

Reply:
Looks good with the new change